assets/Templates/stratcards/pok/0DFFBB93442C459587AD4296177FAE6C.json

```diff
@@ -22,10 +22,10 @@
 		"G": 0,
 		"B": 0
 	},
-	"Flippable": false,
+	"Flippable": true,
 	"AutoStraighten": false,
 	"ShouldSnap": true,
-	"ScriptName": "",
+	"ScriptName": "gameobjects/strategy-card.js",

```

Comment:
shouldn't have a script anymore

the Metadata here is "tile.strategy:pok/construction", keeping the ":base" version should someone choose to use it.